webapp/src/js/services/select2-search.js

```diff
@@ -182,14 +184,10 @@ angular.module('inboxServices').factory('Select2Search',
                         e.params.data.id;
 
             if (docId) {
-              getDoc(docId)
-                .then(function(doc) {
-                  selectEl.select2('data')[0].doc = doc;
-                  selectEl.trigger('change');
-                })
-                .catch(function(err) {
-                  $log.warn('Unable to hydrate select2 selection', err);
-                });
+              getDoc(docId).then(function(doc) {
+                selectEl.select2('data')[0].doc = doc;
+                selectEl.trigger('change');
+              });
```

Comment:
I think we still need to catch and log the unexpected errors here and at line 144. Uncaught promise rejections are really ugly.